src/SFA.DAS.Authorization.Mvc.UnitTests/SFA.DAS.Authorization.Mvc.UnitTests.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <LangVersion>latest</LangVersion>
-        <TargetFrameworks>netcoreapp2.0;net462</TargetFrameworks>
+        <TargetFramework>net462</TargetFramework>
```

Comment:
Why not target multiple frameworks. Is it because it fails on the build server?

Reply:
The assembly under test, `SFA.DAS.Authorization.Mvc` only targets net462, so it doesn't make sense to unit test it using netcoreapp2.0.
The description of this PR details what runs successfully on the build server, and what doesn't, and the reasons why.